tests/templates/test_integration.py

```diff
@@ -269,7 +273,16 @@
 
 class TestIntegrationQnode:
     """Tests the integration of templates into qnodes when differentiable arguments are passed as
-    primary or auxiliary arguments to the qnode, using different interfaces."""
+    primary or auxiliary arguments to the qnode, using different interfaces.
+
+    "Differentiable arguments" to a template are those that in principle allow a user to compute gradients for,
```

Comment:
```suggestion
    "Differentiable arguments" in a template are those that in principle allow a user to compute gradients for,
```
Actually I'm not sure, maybe the original version is fine too :thinking: